hash/m20.xml

```diff
@@ -337,23 +399,36 @@ license:CC0-1.0
 		</part>
 	</software>
 
-	<software name="olitest">
-		<description>OliTest</description>
-		<year>19??</year>
+	<software name="olitest"><!-- bootable pcos20 -->
+		<description>OliTest (Copy 1)</description>
+		<year>1981</year>
 		<publisher>Olivetti</publisher>
 
 		<part name="flop1" interface="floppy_5_25">
-			<dataarea name="flop" size="280897">
+			<dataarea name="flop" size="286720">
 				<rom name="olitest.imd" size="280897" crc="f5e7509b" sha1="994647d937b179eec7b6bb5f157646692bd6b5ba"/>
 			</dataarea>
 		</part>
 	</software>
 
-	<software name="olitutor" supported="no">
-		<description>OliTutor (Ita)</description>
+	<software name="olitesta" cloneof="olitest"><!-- bootable (pcos20) -->
```

Comment:
I did some more digging and discovered that this image is actually a duplicate/ identical copy of olitest(imd). The checksum of the unpadded data part is identical. Since both versions are floating around, what's the guideline for this? I am inclined to remove the duplicate and maybe just leave a comment with the original.

Reply:
Remove this, since it is a 100% duplicate in a different format (img/ imd).